tests/views_wpunit/Tribe/Events/Views/V2/Template_BootstrapTest.php

```diff
@@ -245,6 +247,62 @@ public function should_not_filter_a_404_template() {
 		$this->assertEquals( 'foo/bar.php', $bootstrap->filter_template_include( 'foo/bar.php' ) );
 	}
 
+
+	/**
+	 * It should return the event template path when unfiltered.
+	 *
+	 * @test
+	 */
+	public function it_should_return_the_event_template_path_when_unfiltered() {
+		$default_events = '/var/www/html/wp-content/plugins/the-events-calendar/src/views/v2/default-template.php';
```

Comment:
This path is tightly coupled to the platform running it (`tric` I guess); please change this path to be dynamic in nature and not fail just because it's running on a different machine or if `the-evnents-calendar` directory is called with another name:

```suggestion
		$default_events = \Tribe__Events__Main::instance()->plugin_path . 'src/views/v2/default-template.php';
```

Do the same elsewhere.
I'm using `\Tribe__Events__Main` in this suggested code but, please, alias it to improve readability.